manifests/base/crds/full/argoproj.io_workfloweventbindings.yaml

```diff
@@ -91,6 +91,7 @@ spec:
                                   required:
                                   - key
                                   type: object
+                                  x-kubernetes-map-type: atomic
```

Comment:
Looks like all of the >1000 LoC added to the CRDs is this same line

Reply:
It's a feature of K8s described here: https://github.com/kubernetes/kubernetes/pull/84113 .

It is manipulated here but there is no comments describing why. 

https://github.com/argoproj/argo-workflows/blob/main/hack/crds.go#L65

Reply:
It looks like it might still be necessary per #6350's logic. `x-kubernetes-map-type: atomic` was already used, but this dep change ended up adding it to _more_ places -- why more were added due to this version of `controller-gen` is not entirely clear.